src/shogun/multiclass/KNN.cpp

```diff
@@ -31,7 +31,7 @@ CKNN::CKNN()
 	init();
 }
 
-CKNN::CKNN(int32_t k, CDistance* d, CLabels* trainlab)
+CKNN::CKNN(int32_t k, CDistance* d, CLabels* trainlab, KNN_SOLVER_TYPE knn_solver)
```

Comment:
We should aim for mimicing sklearns setup and default parameters, as I think it is really well tuned